tests/test_input.py

```diff
@@ -1269,6 +1269,24 @@ def test_cv_folds_and_grid_search_folds():
     # ('train', 'train/folds_file_test.csv', 7, None) ->  (None, fold_mapping)
     # ('train', 'train/folds_file_test.csv', 7, True) ->  (None, fold_mapping)
     # ('train', 'train/folds_file_test.csv', 7, False) ->  (None, fold_mapping)
+    # ('evaluate', None, None, None) ->  (None, 3)
```

Comment:
What does (None, fold_mapping) represent?

Reply:
As [this comment](https://github.com/EducationalTestingService/skll/blob/df6248695220b3662001812f1a3a66962450fe4a/tests/test_input.py#L1253) explains (probably not very clearly), that tuple represents the value of `cv_folds` and `grid_search_folds` that we expect to come out once `_parse_config_file()` has run.